synth.py

```diff
@@ -28,42 +29,48 @@
     bazel_target="//google/cloud/oslogin/v1:oslogin-v1-py",
     include_protos=True,
 )
-# pb2's are incorrectly generated into deeper directories, so copy separately into proto/
-s.move(
-    library,
-    excludes=[
-        "nox.py",
-        "setup.py",
-        "README.rst",
-        "docs/index.rst",
-        library / "google/cloud/oslogin_v1/proto/oslogin/**",
-        library / "google/cloud/oslogin_v1/proto/oslogin_v1/**",
-    ],
-)
-s.move(library / "google/cloud/oslogin_v1/proto/**/*.py", "google/cloud/oslogin_v1/proto")
-s.move(library / "google/cloud/oslogin/common/**/*.py", "google/cloud/oslogin_v1/proto")
+
+s.move(library / "google/cloud/oslogin_v1")
+
+s.move(library / "google/cloud/oslogin", excludes=[library / "google/cloud/oslogin/common/**/*"])
+s.move(library / "google/cloud/oslogin/common", "google/cloud/oslogin_v1/common", excludes=[library / "google/cloud/oslogin/common/services"])
+
+s.move(library / "tests/unit/gapic/oslogin_v1")
+
+s.move(library / "scripts/fixup_oslogin_v1_keywords.py")
+
+s.move(library / "docs", excludes=[library / "docs/index.rst", library / "docs/common"])
```

Comment:
Are any docs generated for `oslogin.common` (or is it an empty directory?)

Reply:
Updated the PR to add the docs for `oslogin.common`.